mkdocs.yml

```diff
@@ -621,6 +621,7 @@ nav:
           - organizations/managing-repositories.md
           - organizations/using-coding-standards.md
           - organizations/copying-code-patterns-between-repositories.md
+          - organizations/using-gate-policies.md
```

Comment:
I think I understand the logic for sorting the sidebar like this, but having the "copy code patterns" page between the coding standards and gate policies is very jarring.

Until we drop the "copy code patterns" feature, it can be best to have the gate policies page appear first - this way it also matches the order of the tabs under the **Policies** sidebar entry.